flink-end-to-end-tests/flink-end-to-end-tests-common-kafka/src/test/java/org/apache/flink/tests/util/kafka/SQLClientSchemaRegistryITCase.java

```diff
@@ -60,25 +62,32 @@
 /** End-to-end test for SQL client using Avro Confluent Registry format. */
 @Category(value = {TravisGroup1.class})
 public class SQLClientSchemaRegistryITCase {
+    private static final Logger LOG = LoggerFactory.getLogger(SQLClientSchemaRegistryITCase.class);
+    private static final Slf4jLogConsumer LOG_CONSUMER = new Slf4jLogConsumer(LOG);
+
     public static final String INTER_CONTAINER_KAFKA_ALIAS = "kafka";
     public static final String INTER_CONTAINER_REGISTRY_ALIAS = "registry";
     private static final Path sqlAvroJar = TestUtils.getResource(".*avro.jar");
     private static final Path sqlAvroRegistryJar = TestUtils.getResource(".*avro-confluent.jar");
     private static final Path sqlToolBoxJar = TestUtils.getResource(".*SqlToolbox.jar");
     private final Path sqlConnectorKafkaJar = TestUtils.getResource(".*kafka.jar");
 
-    public final Network network = Network.newNetwork();
+    @ClassRule private static final Network network = Network.newNetwork();
 
     @ClassRule public static final Timeout TIMEOUT = new Timeout(10, TimeUnit.MINUTES);
 
-    @Rule
-    public final KafkaContainer kafka =
-            new KafkaContainer(DockerImageName.parse(DockerImageVersions.KAFKA))
-                    .withNetwork(network)
-                    .withNetworkAliases(INTER_CONTAINER_KAFKA_ALIAS);
+    @ClassRule
+    private static final KafkaContainer kafka =
+            new KafkaContainer(DockerImageName.parse(DockerImageVersions.KAFKA)) {
+                @Override
+                protected void doStart() {
+                    super.doStart();
+                    this.followOutput(LOG_CONSUMER);
+                }
+            }.withNetwork(network).withNetworkAliases(INTER_CONTAINER_KAFKA_ALIAS);
```

Comment:
Can we use `.setLogConsumers()` to set the `LOG_CONSUMER`? It seems overriding `doStart` is a little trick to me.

Reply:
Hi wuchong, thanks a lot for your review. I have changed the overriding to `withLogConsumer()` for readability. 
Let me know if it is better.